packages/mobile/src/escrow/saga.ts

```diff
@@ -88,6 +90,7 @@ function* transferStableTokenToEscrow(action: EscrowTransferPaymentAction) {
     yield put(fetchSentEscrowPayments())
   } catch (e) {
     Logger.error(TAG + '@transferToEscrow', 'Error transfering to escrow', e)
+    CeloAnalytics.track(CustomEventNames.escrow_failed_to_transfer, { error: e.message })
```

Comment:
Do we want to be using Analytics as the channel for tracking errors? It seems like Sentry is already the goto place for that. We can send events to sentry manually if we want

Reply:
Analytics probably shouldn't be the only place but I do think it's useful to track here.

Sentry could potentially be configured to serve some of the same needs but I believe this gives us an easy way to contextualize errors with general app usage and group and analyze them the way we want. I imagine this won't be heavily relied upon for debugging.

Reply:
I'm not working on the wallet day to day of course, but my 2cents are that indeed they should be sent via analytics. In a lot of places in the mobile wallet we are catching errors which makes using analytics data very challenging because it's virtually impossible to tell what has happened without knowing what kind of errors occurred.

That being said, what I'd say is more important is for the mobile wallet team to decide what the various outlets (device logs, sentry, segment analytics) are supposed to do. Maybe we aren't supposed to use analytics to analyze how users are moving through the app?

Reply:
I'm _okay_ with us using analytics for this but skeptical we'll made good use of that data given that we don't check it. We do check Sentry. I'd rather us use one channel well than use both half-heartedly.

Reply:
+1 to Tarik on this not being the only place to track, but a useful place for us to contextualize the errors

@jmrossy I agree that Sentry is the best place for us to first become aware of errors as we check it regularly (the errors will still be piped to Sentry), but it is also helpful to have access to them in our Analytics to see what other actions occurred leading up to the error. This has been helpful while debugging verification flakiness - our current analytics only show that a generic "Verification Failed" error message was thrown, but while looking into different types of failures in the analytics we'd like to be able to see more details on that error